test/samples/consts_simple.instrumented.sol

```diff
@@ -0,0 +1,49 @@
+/// This file is auto-generated by Scribble and shouldn't be edited directly.
+/// Use --disarm prior to make any changes.
+pragma solidity 0.8.13;
+
+/// Utility contract holding a stack counter
+contract __scribble_ReentrancyUtils {
+    event AssertionFailed(string message);
+
+    event AssertionFailedData(int eventId, bytes encodingData);
+
+    bool __scribble_out_of_contract = true;
+    /// Implementation of user constant const uint256 H := (60 * 60)
+    uint256 public constant H_15_0 = 60 * 60;
+    /// Implementation of user constant const uint256 D := (H * 24)
+    uint256 public constant D_15_0 = H_15_0 * 24;
+    /// Implementation of user constant const uint256 W := (D * 7)
+    uint256 public constant W_15_0 = D_15_0 * 7;
+}
+
+/// #const uint256 H := 60 * 60;
+///  #const uint256 D := H * 24;
+contract Some is __scribble_ReentrancyUtils {
+    function testHD() public {
+        _original_Some_testHD();
+        unchecked {
+            if (!(D_15_0 == (H_15_0 * 24))) {
+                emit AssertionFailed("2: ");
+                assert(false);
+            }
+        }
+    }
+
+    function _original_Some_testHD() private {}
+}
+
+/// #const uint256 W := D * 7;
```

Comment:
Hmmm, do we have a guarantee that contracts in a single file will be processed by scribble in the same order as they are defined in the file? I.e. is it possible for scribble to process `Other` before processing `Some`?

Reply:
I would say yes, except if `sortContracts()` during `tcUnits()` will be involved somehow. But I did discovered nay cases when this actually happends... Declating base contract _after_ inheriting in single file is restricted by compiler since 0.4.13. The following source can not even be compiled (tested in Remix IDE):

```solidity
/// #const bool W := D * 7;
contract Other is Some {
    /// #if_succeeds W == D * 7;
    function testWHD() public {}
}

/// #const uint256 H := 60 * 60;
/// #const uint256 D := H * 24;
contract Some {
    /// #if_succeeds D == H * 24;
    function testHD() public {}
}
```

Reply:
Except the inheritance, Scribble do not have any other reasons to change processing order of contracts.

Reply:
This makes sense. I think the logic is ok then